### ISSUE TITLE
Service update notification bar constantly reappearing

### ISSUE THREAD

**Issue by wpigoury** [meetfranz/franz#1129]
### Expected Behavior
Service update notification bar should only appear when a service has been updated.

### Current Behavior
The service update notification bar always reappear after some minutes even after reloading the services.

### Screenshots (if appropriate):
![franz_issue](https://user-images.githubusercontent.com/6941240/48908011-6fd4e480-ee69-11e8-8312-019a5d77c51b.PNG)

### Possible Solution
No idea. Maybe it's just that some services are indeed constantly updated but I seriously doubt that any of the services has been updated that often in the last days.

### Steps to Reproduce (for bugs)
1. Launch Franz
2. Wait a minute or 2
3. Service update notification bar appears
4. Reload services
5. Wait a minute or 2
6. Service update notification bar reappears

### Context
Just using Franz.

### Your Environment
* Franz Version used: 5.0.0-beta.18
* Franz Release used: 1.8.4 / win32 / x64
* Operating System and version: Windows 10 Professional Version 1709
* Active services: Slack, LinkedIn, Facebook Messenger, Trello, Android Message (custom)

**Comment by dzeidzei** [comment]
Just to confirm this is not unique case. I've had this same exact problem now day or two also.

Franz Version used: 5.0.0-beta.18
Operating System and version: Windows 8.1
Active services: Gmail, Hangouts, Android Message

**Comment by BrandonMorr** [comment]
+1 to this, have been seeing this notification pop up every 5-10 minutes.

**Comment by BrandonMorr** [comment]
+1 to @Cahl-Dee's suggestion, haven't seen any notifications since I removed my custom Android Messages.

**Comment by abmantis** [comment]
I don't have any custom service and I'm having this issue. I'm using Messenger, Slack, Discord and WhatsApp.

**Comment by mennorenkens** [comment]
I had the same issue. Turned out I had WhatsApp in both my normal recipes as well as my dev-recipes (for testing purposes). Removing the one from dev fixed the problem.

**Comment by carlba** [comment]
Yes the dev recipes seems to be the culprit here. Mine was the Android Messages one.

**Comment by RicardoMadaleno** [comment]
I don't have any dev recipes but I'm still getting the service update notification bar every 2 or 3 minutes...

it doesn't seem to be an issue with the dev recipes alone... @abmantis was also having the issue without and custom services...
> I don't have any custom service and I'm having this issue. I'm using Messenger, Slack, Discord and WhatsApp.

is anyone else still having this issue?
